### ISSUE TITLE
[Feature Request] Option to activate background clipboard text monitoring from the search page

### ISSUE THREAD

**Issue by psychodelicwasabi** [FooSoft/yomichan#1855]
Just like the usual clipboard monitoring, I'd like to also have the background monitoring option in the search page.

Since this option creates a search page or focus an opened one every time Japanese text is copied, I can't have it enabled all the time, but use it whenever I'm using an OCR to read. However, having to open the settings every time I plan to use it is slightly painful, when I'm used to just pressing alt + insert and quickly open the search page whenever I need it. I wonder if there's anything that could be done, thank you.

**Comment by toasted-nutbread** [comment]
Would it suit your needs if the existing background clipboard scanner had an option to disable focusing the search page when scanning a new term, if the window is already open?

**Comment by psychodelicwasabi** [comment]
No, that's exactly what I want it to do. What I'm requesting is a quicker way to do it without having to open the settings to enable the background scanner every time I intend to do specifically that. Sorry for the lack of clarity.

**Comment by toasted-nutbread** [comment]
As a potential short-term solution, one way to do it could be to set up a second profile with background scanning, and switch the active profile via keyboard shortcut or via the Yomichan button.

**Comment by toasted-nutbread** [comment]
One additional option is that you can set up a keyboard shortcut with action _Toggle option_ and _Path_ = `clipboard.enableBackgroundMonitor`, and that hotkey will toggle the option on/off.

**Comment by psychodelicwasabi** [comment]
I thought of having a shortcut for that, but had no idea how to do it. However, the profile option completely escaped me. I'll try to get used to it, thank you.

**Comment by toasted-nutbread** [comment]
For the shortcut, you can add it on the settings page. It would look like this after adding a new shortcut:

![image](https://user-images.githubusercontent.com/11037431/127693443-4467056c-d9a7-4ff8-a5df-9d4636c32060.png)

**Comment by psychodelicwasabi** [comment]
Great, thank you.
